core/src/main/java/io/questdb/griffin/engine/functions/catalogue/MatViewsFunctionFactory.java

```diff
@@ -288,13 +297,14 @@ public void of(
             metadata.add(new TableColumnMetadata("view_name", ColumnType.STRING));
             metadata.add(new TableColumnMetadata("refresh_type", ColumnType.STRING));
             metadata.add(new TableColumnMetadata("base_table_name", ColumnType.STRING));
-            metadata.add(new TableColumnMetadata("last_refresh_timestamp", ColumnType.TIMESTAMP));
+            metadata.add(new TableColumnMetadata("last_refresh_finished_timestamp", ColumnType.TIMESTAMP));
             metadata.add(new TableColumnMetadata("view_sql", ColumnType.STRING));
             metadata.add(new TableColumnMetadata("view_table_dir_name", ColumnType.STRING));
             metadata.add(new TableColumnMetadata("invalidation_reason", ColumnType.STRING));
             metadata.add(new TableColumnMetadata("view_status", ColumnType.STRING));
             metadata.add(new TableColumnMetadata("refresh_base_table_txn", ColumnType.LONG));
             metadata.add(new TableColumnMetadata("base_table_txn", ColumnType.LONG));
+            metadata.add(new TableColumnMetadata("last_refresh_started_timestamp", ColumnType.TIMESTAMP));
```

Comment:
nit: `last_refresh_start_timestamp`

Reply:
Done in c25a3f258d7a18de505f891e25ebd5c8a600e66a